ada/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+# Create your models here.
+
+
+class RedirectUrl(models.Model):
+
+    urls = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Shouldn't this just be `url`, since we're putting a single URL here? Also, we can probably use django's `URLField` here instead for some additional validation